src/sentry/quotas/base.py

```diff
@@ -26,6 +27,20 @@ def api_name(self):
         return self.name.lower()
 
 
+@dataclass
+class AbuseQuota:
+    id: str
+    option: str
+    # `compat_options` were previously present in getsentry
+    # for errors and transactions. The first one is the org
+    # option for overriding the global option, the second one.
```

Comment:
The second one is what?

Reply:
Good question, I have no idea, I copied the comment from the function to the field because it felt more appropriate